notary-server/notary-server.Dockerfile

```diff
@@ -0,0 +1,39 @@
+# !!! To use this file, please run docker run at the root level of this repository
```

Comment:
Need to do this because we need to copy in dependency files from `tlsn` and `components` — mainly due to the restriction of docker build context